### DISCUSSION TITLE
Convert datetime string from `mm/dd/yyyy HH:MM:SS AM/PM` to timestamp using bloblang

### DISCUSSION THREAD

**Discussion by jean-altura** [redpanda-data/connect#2799] [Q&A]
Hello everyone,

Has anyone successfully converted a datetime string in the format mm/dd/yyyy HH:MM:SS AM/PM to a timestamp using Bloblang? Any insights or examples would be greatly appreciated.

Thanks!

**Comment by mihaitodor** [comment]
Hey @jean-altura 👋 Try this: `root = "08/22/2024 12:54:18 PM".ts_strptime("%m/%d/%Y %H:%M:%S %p")`
